### PR TITLE
add example project + migrate to WKWebView

### PR DESCRIPTION
**Pull request by szotp**

### CODE REVIEW

File: plugin.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
   xmlns:rim="http://www.blackberry.com/ns/widgets"
   xmlns:android="http://schemas.android.com/apk/res/android"
   id="com.mwaysolutions.cordova.webviewplugin"
-  version="0.1.2">
+  version="0.2.0">
```
Comment (MathiasTim) [inline plugin.xml]: Please set to `2.0.0`

File: package.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-    "version": "1.0.0",
+    "version": "0.2.0",
```
Comment (MathiasTim) [inline package.json]: Please set to `2.0.0`

File: plugin.xml (hunk 1 above)
Comment (szotp) [reply plugin.xml]: Done.

File: package.json (hunk 2 above)
Comment (szotp) [reply package.json]: Done.

**Comment by szotp** [comment]
Apperently example projects inside plugin repositories are not supported: https://github.com/apache/cordova-cli/issues/401

I will have to remove it, which is a shame.

**Pull request merged**